.github/workflows/e2e.yaml

```diff
@@ -20,12 +20,13 @@ on:
     types:
       - completed
     branches:
+      - "pull-request/[0-9]+"
```

Comment:
I think we need to make the same change to the images workflow:
```
diff --git a/.github/workflows/image.yaml b/.github/workflows/image.yaml
index fc42721e..b73746b8 100644
--- a/.github/workflows/image.yaml
+++ b/.github/workflows/image.yaml
@@ -16,15 +16,9 @@
 name: Image
 
 on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
   push:
     branches:
+      - "pull-request/[0-9]+"
       - main
       - release-*
 ```

Reply:
added to `.github/workflows/image.yaml` now if we want moving forward, we edit line 50 of that file, to push during PR, but that as a followup